### ISSUE TITLE
Experiment 1

### ISSUE THREAD

**Issue by joelowj** [joelowj/NeuralTest#2]
1) Generate synthetic data for predicate 1 (x^2 + y^2 > z^2)
2) Implement a simple neural network
3) Train neural network to predict the predicate 1 with accuracy of at least 80 %

**Comment by joelowj** [comment]
Did a prof of concept NN on predicate 1.
2 hidden layers with 128 and 32 units respectively,
Learning rate of 0.001,
Number of steps 2000,
Achieved 98.8% on test set.
